priv/repo/migrations/20220524233403_update_sections_add_cover_image.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Oli.Repo.Migrations.UpdateSectionsAddCoverImage do
+  use Ecto.Migration
+
+  def change do
+    alter table(:sections) do
+      add :cover_image, :string, null: true
```

Comment:
Shouldn't we define a default here?

Reply:
Yup, adding.

Reply:
If we define a default, does it makes sense the `null: true`?

Reply:
I was going for `null: true, default: null`. Thinking about the new sections parting from products that do not have a `cover_image` at the moment. To allow them but also set them by default